### DISCUSSION TITLE
(Scala 3.4.0) Should I worry about this?

### DISCUSSION THREAD

**Discussion by spamegg1** [VirtusLab/scala-cli#2767] [General]
![Screenshot from 2024-03-02 20-19-10](https://github.com/VirtusLab/scala-cli/assets/4255997/0ab156b2-b347-425f-869c-4d5b217fab3e)

1.1.3 is the latest right now!

**Comment by Gedochao** [comment]
This is tracked in https://github.com/VirtusLab/scala-cli/issues/2768
